pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "genome-browser"
+version = "0.1.1"
+description = "A lightweight Python 3 graphing API for genomic intervals and genomic features."
+authors = ["Clint Valentine <valentine.clint@gmail.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = ">=3.7,<3.11"
+matplotlib = "^3.5.2"
+more-itertools = "^8.13.0"
+scipy = "^1.7.3"
```

Comment:
`No module named...` + this is the latest `scipy` that doesn't retire the trusty Python 3.7.